src/app-icons.html

```diff
@@ -14,13 +14,14 @@
       <g id="facebook"><path d="M17.6 7.7h-3.8V5.3c0-0.9 0.6-1.1 1-1.1 0.4 0 2.7 0 2.7 0V0h-3.7c-4.1 0-5 3.1-5 5v2.7H6.5V12h2.4c0 5.4 0 12 0 12h5c0 0 0-6.6 0-12h3.4L17.6 7.7z"/></g>
       <g id="github"><path d="M11.9 0.4C5.3 0.4 0 5.7 0 12.3c0 5.3 3.4 9.7 8.1 11.3C8.7 23.7 9 23.4 9 23.1c0-0.3 0-1.2 0-2.2 -3.3 0.7-4-1.4-4-1.4 -0.5-1.4-1.3-1.7-1.3-1.7 -1.1-0.7 0.1-0.7 0.1-0.7C5 17 5.6 18.2 5.6 18.2c1.1 1.8 2.8 1.3 3.5 1 0.1-0.8 0.4-1.3 0.8-1.6 -2.7-0.3-5.4-1.3-5.4-5.9 0-1.3 0.5-2.4 1.2-3.2C5.5 8.1 5.1 6.9 5.7 5.3c0 0 1-0.3 3.3 1.2 1-0.2 2-0.4 3-0.4s2 0.1 3 0.4c2.3-1.6 3.3-1.2 3.3-1.2 0.6 1.6 0.2 2.9 0.1 3.2 0.8 0.8 1.2 1.9 1.2 3.2 0 4.6-2.8 5.6-5.4 5.9 0.4 0.4 0.8 1.1 0.8 2.2 0 1.6 0 2.9 0 3.3 0 0.3 0.2 0.7 0.8 0.6C20.6 22 24 17.6 24 12.3 23.9 5.7 18.5 0.4 11.9 0.4zM4.5 17.5c0 0-0.1 0.1-0.2 0 -0.1 0-0.1-0.1-0.1-0.2 0 0 0.1-0.1 0.2 0C4.5 17.4 4.6 17.5 4.5 17.5L4.5 17.5zM4.4 17.4L5 18.1c0 0-0.1 0-0.2 0s-0.1-0.2 0-0.2c0 0 0.1 0 0.2 0 0.1 0.1 0.1 0.2 0 0.2l0 0L4.4 17.4zM4.9 17.9l0.6 0.8c-0.1 0-0.2 0-0.3-0.1 -0.1-0.1-0.1-0.2 0-0.3s0.2 0 0.3 0.1 0.1 0.2 0 0.3l0 0L4.9 17.9zM5.5 18.7l0.6 0.7c0 0-0.2 0-0.3 0 -0.1-0.1-0.1-0.2 0-0.3 0 0 0.2 0 0.3 0 0.1 0.1 0.1 0.2 0 0.3l0 0L5.5 18.7zM6.1 19.4L7 19.8c0 0.1-0.1 0.1-0.3 0.1 -0.1 0-0.2-0.1-0.2-0.2 0-0.1 0.1-0.1 0.3-0.1 0.1 0 0.2 0.1 0.2 0.2l0 0L6.1 19.4zM7 19.8l1 0C8 19.9 7.9 20 7.8 20c-0.1 0-0.2-0.1-0.2-0.2 0-0.1 0.1-0.2 0.2-0.2 0.1 0 0.2 0.1 0.2 0.2l0 0L7 19.8zM8 19.9l0.9-0.1c0 0.1-0.1 0.2-0.2 0.2 -0.1 0-0.3 0-0.3-0.1 0-0.1 0.1-0.2 0.2-0.2 0.1 0 0.2 0 0.3 0.1l0 0L8 19.9z"/></g>
       <g id="googlePlus"><path d="M7.6 11c0 0.9 0 1.7 0 2.6 1.5 0 2.9 0 4.4 0 -0.6 3.2-5 4.3-7.3 2.2 -2.4-1.9-2.3-5.9 0.2-7.6 1.7-1.4 4.2-1.1 5.9 0.1 0.7-0.6 1.3-1.3 1.9-2 -1.5-1.2-3.2-2-5.1-1.9C3.7 4.3 0.1 7.7 0 11.7c-0.2 3.2 1.9 6.4 4.8 7.5 2.9 1.2 6.8 0.4 8.7-2.3 1.3-1.7 1.5-3.9 1.4-5.9C12.4 11 10 11 7.6 11zM21.8 11c0-0.7 0-1.5 0-2.2 -0.7 0-1.5 0-2.2 0 0 0.7 0 1.5 0 2.2 -0.7 0-1.5 0-2.2 0 0 0.7 0 1.5 0 2.2 0.7 0 1.5 0 2.2 0 0 0.7 0 1.5 0 2.2 0.7 0 1.5 0 2.2 0 0-0.7 0-1.5 0-2.2 0.7 0 1.5 0 2.2 0 0-0.7 0-1.5 0-2.2C23.2 11 22.5 11 21.8 11z"/></g>
+      <g id="web"><path d="M4.5 11h-2V9H1v6h1.5v-2.5h2V15H6V9H4.5v2zm2.5-.5h1.5V15H10v-4.5h1.5V9H7v1.5zm5.5 0H14V15h1.5v-4.5H17V9h-4.5v1.5zm9-1.5H18v6h1.5v-2h2c.8 0 1.5-.7 1.5-1.5v-1c0-.8-.7-1.5-1.5-1.5zm0 2.5h-2v-1h2v1z"/><path d="M24 24H0V0h24v24z" fill="none"/></g>
```

Comment:
@aljesusg can you update the icon to the ["`http`" `iron-icon`](https://github.com/PolymerElements/iron-icons/blob/master/iron-icons.html#L178)?

```html
<g id="http"><path d="M4.5 11h-2V9H1v6h1.5v-2.5h2V15H6V9H4.5v2zm2.5-.5h1.5V15H10v-4.5h1.5V9H7v1.5zm5.5 0H14V15h1.5v-4.5H17V9h-4.5v1.5zm9-1.5H18v6h1.5v-2h2c.8 0 1.5-.7 1.5-1.5v-1c0-.8-.7-1.5-1.5-1.5zm0 2.5h-2v-1h2v1z"/></g>
```

Thanks!